apps/server/README.md

```diff
@@ -1,16 +1,19 @@
 # Lectern Server
 
-[<img alt="Container Registry Badge" src="https://img.shields.io/badge/GCHR.io-overture--stack%2Flectern-blue?style=for-the-badge&color=blue&cacheSeconds=0" />](https://github.com/overture-stack/lectern/pkgs/container/lectern)
+[<img alt="Container Registry Badge" src="https://img.shields.io/badge/Docker--Image-ghcr.io-blue?style=for-the-badge&color=blue&cacheSeconds=0" />](https://github.com/overture-stack/lectern/pkgs/container/lectern)
+[<img hspace="5" src="https://img.shields.io/badge/chat--with--developers-overture--slack-blue?style=for-the-badge">](http://slack.overture.bio)
+[<img hspace="5" src="https://img.shields.io/badge/License-AGPL--3.0-blue?style=for-the-badge">](https://github.com/overture-stack/lectern/blob/develop/LICENSE)
 
-Lectern Server is the standalone web service for Lectern. It provides an API to create, manage, and share Data Dictionary schemas.
+Lectern Server is a standalone web service for Lectern. It provides an API to create, manage, and share Lectern Dictionary schemas.
```

Comment:
You didn't write this, but `Lectern Server is a standalone web service for Lectern` sounds like explaining an explanation by using the word explanation.

How about?
```suggestion
Lectern can be run as a standalone web service, through the `Server` ExpressJS app. It provides an API to create, manage, and share Lectern Dictionary schemas.
```

Reply:
I'll take half of this, the repetition does read poorly.